vscode/client/src/extension.ts

```diff
@@ -0,0 +1,121 @@
+import * as os from 'os';
+import * as fs from 'fs';
+import { workspace, ExtensionContext } from 'vscode';
+
+import {
+	LanguageClient,
+	LanguageClientOptions,
+	StreamInfo,
+} from 'vscode-languageclient/node';
+
+import { spawn } from 'child_process';
+
+let client: LanguageClient;
+
+export function activate(context: ExtensionContext) {
+	const expandHomeDir = (filePath: string) => {
+		const home = os.homedir();
+		return home ? filePath.replace('~', home) : filePath;
+	};
+
+	// Check if djlsp is installed
+	const djlspPaths = [
+		'~/.local/bin/djlsp',
+		'~/.local/pipx/venvs/django-template-lsp/bin/djlsp',
+	].map(expandHomeDir);
+
+	const djlspPath = djlspPaths.find(fs.existsSync);
+
+	// Add error handling for when djlsp is not installed
+	if (!djlspPath) {
+		throw new Error('djlsp is not installed');
+	}
+
+	const configuration = workspace.getConfiguration('djangoTemplateLsp');
+
+	const djlspArgs = [];
+
+	if (configuration.get("enableLogging", false)) {
+		djlspArgs.push('--enable-log');
+	}
+
+	// Pass initialization options to the server
+	const initializationOptions = {
+		docker_compose_file: configuration.get('dockerComposeFile', 'docker-compose.yml'),
+		docker_compose_service: configuration.get('dockerComposeService', 'django'),
+		django_settings_module: configuration.get('djangoSettingsModule', ''),
+	};
+
+	// Get current workspace folder path
+	// This is also updated when the workspace folder is changed.
+	const workspaceFolders = workspace.workspaceFolders;
+	if (workspaceFolders === undefined) {
+		throw new Error('No workspace folder is open');
+	}
+	const workspaceFolder = workspaceFolders[0];
+	const workspaceFolderPath = workspaceFolder.uri.path;
+
+	// If the extension is launched in debug mode then the debug server options are used
+	// Otherwise the run options are used
```

Comment:
Is this comment still correct? it looks like it uses the enableLogging option

Reply:
No this comment is not correct anymore, it has now been removed.